charts/region/values.schema.json

```diff
@@ -313,12 +313,6 @@
                   "selector": {
                     "type": "object",
                     "properties": {
-                      "properties": {
```

Comment:
Note: this is a breaking change, and will need a release note.

Reply:
What intervention would be needed to upgrade a running system past the change?
Could this be made backward-compatible by keeping it in, but deprecating its use?

Reply:
Well, we're going to have to remove it eventually, I see no difference doing it now compared until later 🤷🏻  If you are insistent I may be persuaded with a tequila.